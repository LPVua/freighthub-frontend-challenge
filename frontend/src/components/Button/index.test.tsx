import React from 'react';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Button } from '.';

Enzyme.configure({ adapter: new Adapter() });

describe('Button', () => {
  it('should render correct DOM', () => {
    const wrapper = shallow(<Button onClick={() => {}}>Value</Button>)
    expect(wrapper).toMatchSnapshot();
  })
})