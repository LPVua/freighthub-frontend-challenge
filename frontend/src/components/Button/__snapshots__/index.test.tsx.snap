// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render correct DOM 1`] = `ShallowWrapper {}`;
